.circleci/scripts/build_announce.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -euox
+
+VERSION=$(go run build/rpm/main.go version)~${CIRCLE_TAG##*-}
+
+SHORT_SHA1=${CIRCLE_SHA1:0:7}
+CIRCLE_PR_NUMBER=$(hub pr list -s open -L 10 -f "%I")
+
+PACKAGES="kcn kpn ken kscn kspn ksen kbn kgen homi"
+BAOBAB_PACKAGES="kcn kpn ken"
+#PACKAGE_PREFIX="http://packages.klaytn.net/klaytn/${VERSION}"
```

Comment:
Is it ok not to use this commented code?

Reply:
you are right. I reflect your comment! thank you for letting me know